package.json

```diff
@@ -3,28 +3,29 @@
   "private": true,
   "workspaces": [
     "packages/@tinacms/*",
+    "deprecated-packages/*",
     "examples/*",
     "experimental-examples/*",
     "packages/[^@]*"
   ],
   "scripts": {
-    "build": "yarn buildBuilder && yarn workspaces foreach --topological-dev --no-private run g:echo | cat > topologicalDeps.txt && yarn workspace @tinacms/scripts build:all",
+    "build": "yarn buildBuilder && yarn workspaces foreach --exclude \"deprecated-packages/*\" --topological-dev --no-private run g:echo | cat > topologicalDeps.txt && yarn workspace @tinacms/scripts build:all",
     "build:e2e": "yarn build && yarn workspace kitchen-sink-starter build",
     "start:e2e": "yarn workspace kitchen-sink-starter start",
     "watch": "yarn build && yarn workspace @tinacms/scripts watch:all",
     "buildBuilder": "yarn workspace @tinacms/scripts build",
     "test": "yarn workspaces foreach --no-private -vip run test",
     "test:dev": "cypress open",
     "test:e2e": "yarn cypress run",
-    "nuke": "yarn workspaces foreach --no-private -vip run g:nuke",
-    "types": "yarn workspaces foreach --topological-dev --no-private -vip run types",
+    "nuke": "yarn workspaces foreach --exclude \"deprecated-packages/*\" --no-private -vip run g:nuke",
+    "types": "yarn workspaces foreach --exclude \"deprecated-packages/*\" --topological-dev --no-private -vip run types",
     "g:echo": "echo $INIT_CWD",
     "g:nuke": "cd $INIT_CWD && yarn exec rm -rf dist && echo \"cleaned\"",
     "version": "yarn changeset version --ignore tina-cloud-starter --ignore tina-tailwind-sidebar-demo --ignore starter-basic --ignore kitchen-sink-starter",
```

Comment:
Does this need to exclude those packages as well?

Reply:
I am having some issues with this PR going to go ahead with https://github.com/tinacms/tinacms/pull/2500#discussion_r792929458